tests/ut/ir/transforms/test_flatten_tile_nd_to_2d.py

```
@@ -2445,22 +2459,22 @@ def test_batch_two_acc_unrolls_without_acc_roundtrip_moves(self):
         names = [c.op.name for c in calls]
 
         # Both batch ops are fully unrolled.
-        assert "tile.batch_matmul" not in names
-        assert "tile.batch_matmul_acc" not in names
+        assert _OP_TILE_BATCH_MATMUL not in names
+        assert _OP_TILE_BATCH_MATMUL_ACC not in names
 
         # Two batches × {matmul, matmul_acc} = 2 + 2.
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)